core/controllers/home.py

```diff
@@ -56,7 +56,7 @@ def get(self):
         """Handles GET requests."""
         if self.user_id is None:
             raise self.PageNotFoundException
-
+        # pylint: disable=line-too-long
```

Comment:
I think you actually need to put this on the right of the relevant line, so that it disables that line and nothing else. If I'm not mistaken, putting it here disables it for all lines going forward. Note that it doesn't matter if, when you put the warning on the right of the line, the line subsequently becomes longer than 80 chars.

Here is an example: https://github.com/oppia/oppia/blob/develop/core/domain/fs_domain.py#L138